pkg/didcomm/protocol/introduce/service.go

```diff
@@ -22,3 +25,34 @@ const (
 	// AckMsgType defines the introduce ack message type.
 	AckMsgType = IntroduceSpec + "ack"
 )
+
+// Service for introduce protocol
+type Service struct {
+	Action
+}
+
+// New returns introduce service
+func New() *Service {
+	return &Service{}
+}
+
+// Handle didexchange msg
+func (s *Service) Handle(msg dispatcher.DIDCommMsg) error {
+	return nil
+}
+
+// Name returns service name
+func (s *Service) Name() string {
+	return Introduce
+}
+
```

Comment:
Service would need [Accept](https://github.com/hyperledger/aries-framework-go/blob/master/pkg/didcomm/dispatcher/api.go#L18) method as well for dispatcher.

Reply:
Done!